ngx_http_access_token_to_jwt.c

```diff
@@ -127,6 +137,20 @@ static ngx_int_t ngx_http_access_token_to_jwt_handler(ngx_http_request_t *reques
         return NGX_DECLINED;
     }
 
+    if (module_location_config->client_secret.len == 0)
+    {
+        ngx_log_error(NGX_LOG_WARN, request->connection->log, 0,
+            "Module not configured properly: missing client secret");
+        return NGX_DECLINED;
+    }
+
+    if (module_location_config->client_id.len == 0)
+    {
+        ngx_log_error(NGX_LOG_WARN, request->connection->log, 0,
+                           "Module not configured properly: missing client id");
+        return NGX_DECLINED;
+    }
+
     ngx_str_t encoded_client_credentials = module_location_config->base64encoded_client_credentials;
 
     if (encoded_client_credentials.len == 0)
```

Comment:
Ya, that's removed at https://github.com/curityio/nginx_introspect/pull/10/files#diff-015a4cf98d13f6f15adc8c7c8584f672L70. Lines 154 to 164 should be removed.